dts/bindings/sensor/ti,fdc2x1x.yaml

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2020 arithmetics ITS
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments FDC2X1X capacitive sensor
+
+compatible: "ti,fdc2x1x"
+
+include: i2c-device.yaml
+
+properties:
+    sd-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The SD pin defaults to active high when consumed by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    intb-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INTB pin defaults to active low when produced by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    fdc2x14:
+      type: boolean
+      description: |
+        Set to identify the sensor as FDC2114 or FDC2214 (4-channel version)
+
+    autoscan:
```

Comment:
Couldn't this just be boolean?

Reply:
Yes a boolean makes more sense. I changed that